tests/bootstrap.php

```diff
@@ -9,3 +9,7 @@
 } else {
     throw new Exception('Can\'t find autoload.php. Did you install dependencies with Composer?');
 }
+
+if ( ! class_exists('PHPUnit\Framework\Error\Warning')) {
+    class_alias('PHPUnit_Framework_Error_Warning', 'PHPUnit\Framework\Error\Warning');
+}
```

Comment:
This ensures that any reference to `PHPUnit\Framework\Error\Warning` will alias to `PHPUnit_Framework_Error_Warning`, which doesn't currently affect anything. The GridFS tests still use `PHPUnit_Framework_Error_Warning`, so they work for PHPUnit 4 but fail in 6.

See the renaming I suggested at the bottom of https://github.com/mongodb/mongo-php-library/pull/489#discussion_r166971931